build/toolchain/gcc_toolchain.gni

```diff
@@ -610,7 +614,12 @@ template("gcc_toolchain") {
       if (using_old_compiler) {
         link_command = "$ld {{ldflags}}${extra_ldflags} -o \"$unstripped_outfile\" $start_group_flag @\"$rspfile\" {{solibs}} $end_group_flag {{libs}} $tail_lib_dependencies"
       } else {
+        # Windows based platform modular builds dont link directly with the shared library.
+        if(is_host_win){
```

Comment:
Doesn't this need to also be conditional on if the build is modular?

Reply:
yup, i can add the sb_is_modular : although atm : this applies only to ps5 , win/xb1 dont use this toolchain

Reply:
added sb_is_modular